sisyphus/job.py

```diff
@@ -516,10 +516,12 @@ def _sis_cleanup(self):
                 try:
                     if not gs.JOB_CLEANUP_KEEP_WORK:
                         shutil.rmtree(os.path.abspath(self._sis_path(gs.JOB_WORK_DIR)))
+                    if not gs.JOB_CLEANUP_KEEP_INPUT:
```

Comment:
I don't like this to be honest. I have had cases in the past where I would unzip the cleaned file to just have the input folder. While I will most likely set the flag, I feel like this can potentially cause spots where information is a lot harder to get (if the info file is weird/bloated).

Reply:
I don't exactly understand the case that the `info` file is weird/bloated. Can you give some example? In my scripts, I just always extract this information automatically from the `info` file, and never had any problems.

Reply:
In a RASR recognition usually the whole RASR config is in the info file. If you have split your data into mulitple hdfs then each of them is also in there, even though they might point to the same job. 
I just personally prefer to be able to cd into the dir (and then easily cd back with ..).